core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -85,15 +84,17 @@ object RuntimeStatus extends Enum[RuntimeStatus] {
   case object Stopping extends RuntimeStatus
   case object Stopped extends RuntimeStatus
   case object Starting extends RuntimeStatus
+  case object Staging extends RuntimeStatus
 
   // A user might need to connect to this notebook in the future. Keep it warm in the DNS cache.
-  val activeStatuses: Set[RuntimeStatus] = Set(Unknown, Creating, Running, Updating, Stopping, Stopped, Starting)
+  val activeStatuses: Set[RuntimeStatus] =
+    Set(Unknown, Creating, Running, Updating, Stopping, Stopped, Starting, Staging)
 
   // Can a user delete this runtime? Contains everything except Creating, Deleting, Deleted.
   val deletableStatuses: Set[RuntimeStatus] = Set(Unknown, Running, Updating, Error, Stopping, Stopped, Starting)
 
   // Non-terminal statuses. Requires monitoring via ClusterMonitorActor.
-  val monitoredStatuses: Set[RuntimeStatus] = Set(Unknown, Creating, Updating, Deleting, Stopping, Starting)
+  val monitoredStatuses: Set[RuntimeStatus] = Set(Unknown, Creating, Updating, Deleting, Stopping, Starting, Staging)
```

Comment:
we're introducing a new status? can we have newline for each of the status and add some comment about what each status means?

Reply:
Hm, `RuntimeStatus` is kind of overloaded. It's used for both _Leo_ runtime status and _Google_ status for the Dataproc cluster/GCE instance. I added `Staging` because that's a Google status for GCE, but we never actually update the Leo runtime status to `Staging` (we still use `Creating`). So all this change does is prevent `ClusterMonitorActor` from logging "Unknown status" log messages.

I think there's some work to be done to revisit our statuses and monitoring code. I'd prefer to do that in a separate PR though.

Reply:
+1 that when we revisit statuses, it'd be clarifying to separate our statuses from Google statuses.

Reply:
Yeah, maybe could be done as part of bullet 2 in https://broadworkbench.atlassian.net/browse/IA-1747

Reply:
yea...I was planning to address this as part of that ticket

Reply:
👍 Added a note to that ticket.